public/index.html

```diff
@@ -270,16 +87,114 @@ <h1>
                 </div>
               </div>
             </form>
-            <div className="card">
-              <div className="refresh-wrapper">
-                <button type="button" className="refresh">
-                  <img src="refresh.svg" />
-                </button>
+            <section class="flex-box">
```

Comment:
React uses `className` instead of `name`
```suggestion
            <section className="flex-box">
```